protocol/src/index.ts

```diff
@@ -651,6 +651,21 @@ export namespace Protocol {
     '|controlshtml|': readonly ['controlshtml', HTML];
     '|fieldhtml|': readonly ['fieldhtml', HTML];
     '|debug|': readonly ['debug', Message];
+    '|deinit|': readonly ['deinit'];
+    '|selectorhtml|': readonly ['selectorhtml', string, HTML];
```

Comment:
I think we might need to introduce a `SelectorName` type similar to how `UHTMLName` type is declared above? Unless I'm misunderstanding what the second arg is used for.

Reply:
The string arg definitely looks like a CSS selector. Should be fine to just say `SelectorName` and leave a doc comment about it.